### PR TITLE
Move asset apps into repo and introduce lerna for build

### PR DESCRIPTION
**Pull request by lehnerm**

### CODE REVIEW

**Comment by davidfateh** [comment]
Would it be possible for you to point out the key changes in this PR? It's going to take a lot of time to sift through it manually otherwise.

File: apps/bynder/.coverage/lcov-report/base.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
+body, html {
```
Comment (jelz) [inline apps/bynder/.coverage/lcov-report/base.css]: Don't commit coverage reports

File: apps/bynder/MARKETPLACE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Integrate with [Bynder's](https://www.bynder.com) digital asset management.
```
Comment (jelz) [inline apps/bynder/MARKETPLACE.md]: Don't include MARKETPLACE files files

File: apps/bynder/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## Bynder Assets
```
Comment (jelz) [inline apps/bynder/README.md]: I'd just create a simple README per app pointing to the docs. This file mentions UI Extensions and is outdated

File: apps/cloudinary/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (jelz) [inline apps/cloudinary/tsconfig.json]: cloudinary is JS, remove this file

File: apps/smartling/lambda/src/index.ts [hunk 5]
```diff
@@ -107,11 +107,7 @@ export function makeApp(fetchFn: any, issuer: any) {
     );
   });
 
-  if (process.env.LOCAL_DEV === 'true') {
-    app.use('/frontend', express.static(path.resolve(__dirname, '../../frontend/build')));
-  } else {
-    app.use('/frontend', express.static(path.resolve(__dirname, '../../static')));
-  }
+  app.use('/frontend', express.static(path.resolve(__dirname, '../node_modules/smartling-frontend/build')));
```
Comment (jelz) [inline apps/smartling/lambda/src/index.ts]: why removed? could be still used in development

File: package.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "apps",
```
Comment (jelz) [inline package.json]: add `private: true`, `author`, `license` fields

File: apps/smartling/lambda/src/index.ts (hunk 5 above)
Comment (lehnerm) [reply apps/smartling/lambda/src/index.ts]: no with `lerna bootstrap` you have it symlinked in node_modules in dev as well

**Pull request merged**